src/orange/surf/Involute.hh

```diff
@@ -163,37 +176,38 @@ CELER_FUNCTION SignedSense Involute::calc_sense(Real3 const& pos) const
 
     real_type const x = pos[0] - origin_[0];
     real_type const y = pos[1] - origin_[1];
-    
+
     /*
      * Calculate distance to origin and obtain t value for disance.
      */
-    real_type const rxy2 = ipow<2>(x)+ ipow<2>(y);
-    real_type const tPoint2 = (rxy2/ipow<2>(r_b_))-1;
+    real_type const rxy2 = ipow<2>(x) + ipow<2>(y);
+    real_type const tPoint2 = (rxy2 / ipow<2>(r_b_)) - 1;
     real_type const tPoint = sqrt(tPoint2);
 
-     /*
-      * Check if Point is on involute.
-      */
+    /*
+     * Check if Point is on involute.
+     */
     real_type const angle = tPoint + a_;
     real_type const xInv = r_b_ * (std::cos(angle) + tPoint * std::sin(angle));
     real_type const yInv = r_b_ * (std::sin(angle) - tPoint * std::cos(angle));
 
-    if (abs(x-xInv) < tol && abs(y-yInv) < tol) { 
-       return SignedSense::on;
-     }
+    if (abs(x - xInv) < tol && abs(y - yInv) < tol)
+    {
+        return SignedSense::on;
+    }
 
     /*
-     * Check if point is in defined bounds. 
+     * Check if point is in defined bounds.
      */
-    if (abs(tPoint2) < ipow<2>(tmin_) - tol) {
-       return SignedSense::outside;
+    if (abs(tPoint2) < ipow<2>(tmin_) - tol)
```

Comment:
We *really really* should not have tolerances when evaluating senses and distance to boundary. When crossing any surface, involute or not, the sense is updated analytically (i.e we flip a bit and *don't* recalculate senses near a surface).  You should never need a tolerance because we will always know whether we are inside, outside, or on.